spec/rubocop/config_spec.rb

```diff
@@ -603,28 +628,46 @@
 
     let(:loaded_path) { 'example/.rubocop.yml' }
 
-    it 'returns true when Include config only includes regular paths' do
-      configuration['AllCops'] = { 'Include' => ['**/Gemfile', 'config/unicorn.rb.example'] }
+    context 'when Include config only includes regular paths' do
+      let(:hash) do
+        { 'AllCops' => { 'Include' => ['**/Gemfile', 'config/unicorn.rb.example'] } }
+      end
 
-      expect(configuration).not_to be_possibly_include_hidden
+      it 'returns true' do
+        expect(configuration).not_to be_possibly_include_hidden
+      end
     end
```

Comment:
Because `InconsistentConfig` uses `@config.for_cop`, configuration is loaded prior to these tests running, so `configuration['AllCops']` doesn't actually change anything. I updated the cops in this file to set the config first which makes the cop run. I'm not sure if there's a way I can get around this but I also don't think this should affect actual rubocop running (ie. I think this is a quirk of how config is loaded in these tests).